index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="public/styles.css">
+  <meta charset="UTF-8">
+  <title>Bowling Scorecard</title>
+</head>
+<body>
+  <section id="form">
+    <form id="bowl">
+      <label for="pins-bowled">Pins bowled:</label>
+      <input id="pins-bowled" type="number" min="0" max="10">
+      <input type="submit" value="Add bowl">
+    </form>
+    <h3><span id="error"></span></h3>
+  </section>
+
+<!--  table format taken from an online example -->
+  <div id='scorecard'>
+    <table id='scorecardTable' class='scorecard' cellpadding='1' cellspacing='0'>
```

Comment:
Would be nice to have buttons for each roll that disappeared depending on whether that option was available.